Dockerfile.desktop

```diff
@@ -8,15 +8,16 @@ USER root
 
 ARG RESOLUTION="1920x1080x24"
 ARG XARGS=""
+ARG FIREFOX_VERSION=""
 
 # Set env
 ENV DEBIAN_FRONTEND=noninteractive \
     LC_ALL=C.UTF-8 \
     LANG=C.UTF-8 \
-    FIREFOX_VERSION=123.0 \
+    FIREFOX_VERSION_NUM=123.0 \
```

Comment:
What's this change?
Only instance of `FIREFOX_VERSION_NUM` I see in this PR.

Reply:
Ah, this can be reverted

Reply:
Wait, this can just be removed